client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -54,6 +73,7 @@ function mapDispatchToProps( dispatch ) {
 		{
 			editProduct,
 			editProductAttribute,
+			dispatchFetchProductCategories: fetchProductCategories,
```

Comment:
If we rename the action creator as suggested above, I _think_ this simplifies as well.